### PR TITLE
Add inputs to build workflows

### PR DESCRIPTION
**Pull request by aaroncameron-wk**
#### Reason for change
#286 

#### Steps to Test
Run manually with various inputs and confirm version/ref inputs are respected

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by sagesmith-wf — DISMISSED** [review]

**Review by derekgengenbacher-wf — DISMISSED** [review]

**Review by austinmatherne-wk — DISMISSED** [review]

File: .github/workflows/build-windows.yml [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+        description: 'Workiva/ixbrl-viewer branch, tag or SHA to checkout (blank for default)'
+        required: false
+        type: string
+      node_version:
+        default: lts/*
+        description: 'Node.js version to use'
+        required: true
+        type: string
+      python_version:
+        default: '3.9'
```
Comment (austinmatherne-wk) [inline .github/workflows/build-windows.yml:20]: The windows workflow [has a few](https://github.com/Arelle/Arelle/blob/211fb1ba912058f8401c09a93a2cc804d79ae92a/.github/workflows/build-windows.yml#L76) references [to the python version](https://github.com/Arelle/Arelle/blob/211fb1ba912058f8401c09a93a2cc804d79ae92a/.github/workflows/build-windows.yml#L79). Can you update those to use the matrix value?

There's additional work to make the windows build scripts actually work with other python versions, but you don't need to worry about getting that to work here.

**Review by sagesmith-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

File: .github/workflows/build-windows.yml (hunk 1 above)
Comment (aaroncameron-wk) [reply .github/workflows/build-windows.yml:20]: Just noticed an issue with using patch versions. The build directory gets named based on the <MAJOR>.<MINOR> version without the patch #. Is that what you're referring to?

Comment (austinmatherne-wk) [reply .github/workflows/build-windows.yml:20]: There's also some stuff with the installer. That doesn't need to be addressed here. The main goal of this is to get the builds working with alternative plugin branches.

**Pull request merged**